pkg/apis/capacityscheduling/register.go

```diff
@@ -0,0 +1,5 @@
+package capacityscheduling
```

Comment:
As "capacity scheduling" is a generic concept, but here we should name the package with a concret API. So let's  name the pkg as "elasticquota" or "elasticresourcequota".

Reply:
Because the group is `scheduling.sigs.k8s.io`, so I rename the  package  as `scheduling`.